sklearn/metrics/pairwise.py

```diff
@@ -231,54 +241,37 @@ def euclidean_distances(X, Y=None, Y_norm_squared=None, squared=False,
     """
     X, Y = check_pairwise_arrays(X, Y)
 
-    # If norms are passed as float32, they are unused. If arrays are passed as
```

Comment:
You've managed to revert several changes to euclidean distances and possibly other parts of the file. Do you need help fixing it up?